applications/blockchainia.md

```diff
@@ -0,0 +1,139 @@
+| **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. Unit tests will be written for relevent game server functionality. Unit tests will be written for the DeathToll pallet. In each guide (in each project's Github repo), we will describe how to run these tests. Integration testing will be enabled through game functionality (i.e, Given an enemy is eliminated on server, the proper state functions are executed on chain, and the leaderboard updates through the in-game chain browser) |
+| **0d.** | Docker | We will provide Dockerfiles that can be used to run and test all the functionality delivered with this milestone. This will include a deployable "Full Node" that encases both a game server and substrate node. These will interact with our game client and substrate parachain respectively. A successful MVP will allow a game client to register events on the game server (Server-Authoritative model), which is then written to the parachain ledger. Upon completion of the initial game "level", an "accolade" NFT will be deposited via ink! smart contract to the wallet that completed the required in-game tasks. |
+| 1. | Social Media | We will deploy a social media marketing campaign upon the successful development and testing of an MVP. This will include a social media campaign, building a discord community of evangelists, and marketing materials that describe our project to potential stakeholders and various customers of our ecosystem. The first milestone will involve hitting 1000 followers, which will enable us to involve our community from Milestone 2 forward. 
+| 2. | Substrate module: DeathToll Game Events | The DeathToll Game Events Substrate module will add functionality for starting/ending a "match"", start and end a "round", eliminate a player, score a point, complete a "task". Tasks examples include "move through a defined space", eliminate an AI enemy, retrieve an item, etc. This deliverable will include unit tests and instructions in the git repository. It is highly likely that we will break this into two pallets, one for core gaming functionality, and one specific to the needs of the DeathToll game-type. 
+| 3. | Unity Game/Server | Both DeathToll Modules of our custom chain will interact with with open-source scrips we create to control our Unity created game server.  Each of these processes will initially need to be deployable on the same machine via docker container, until appropriate configuration is added. 
+| 4. | Community Documentation | This project includes creating the documentation that will allow our community to test and use our infrastructure. We hope to receive feedback on this documentation in Milestone 2.
```

Comment:
_0b. | Documentation_ already contains this, correct? If so, feel free to remove deliverable _4. Community Documentation_.